src/lib/core/CHIPConfig.h

```diff
@@ -1496,6 +1496,18 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_ICD_CLIENTS_SUPPORTED_PER_FABRIC 2
 #endif
 
+/**
+ * @def CHIP_CONFIG_SYNCHRONOUS_REPORTS_ENABLED
+ *
+ * @brief Controls whether the synchronized report scheduler is used.
+ *
+ * The use of the syncrhonous reports feature aims to reduce the number of times an ICD needs to wake up to emit repots to its
```

Comment:
```suggestion
 * The use of the synchronous reports feature aims to reduce the number of times an ICD needs to wake up to emit reports to its
```
though emitting "repots" would be interesting too, esp for an irrigation setup or something.